### PR TITLE
Wage gap

### PR DESCRIPTION
**Pull request by KingTerrytheTerrible**
Women only get paid 70% as much as men

### CODE REVIEW

**Comment by TrickyDickRichard** [comment]
thank you "King Terry" very epic +1

**Comment by zamolxius** [comment]
fucking genius +1
next up make nonhumans not get paid due to being indentured slaves

**Comment by mkikt4743** [comment]
+1 very cool

**Comment by karmaisblackandbluepilled** [comment]
+1 the hero that hippie deserves

File: code/modules/economy/account.dm [hunk 1]
```diff
@@ -50,6 +52,8 @@
 
 /datum/bank_account/proc/payday(amt_of_paychecks, free = FALSE)
 	var/money_to_transfer = account_job.paycheck * amt_of_paychecks
+	if(account_holder_gender == FEMALE)
+		money_to_transfer = money_to_transfer * 0.7
```
Comment (yoyobatty) [inline code/modules/economy/account.dm]: money_to_transfer *= 0.7 instead please

Comment (KingTerrytheTerrible) [reply code/modules/economy/account.dm]: i wasn't sure if BYOND  supported it, thanks

**Comment by QuarianCommando** [comment]
+1

**Comment by diegoxt45** [comment]
why

**Comment by Manly-Dwarf** [comment]
such a daring PR
maybe this will expose the real problems with society to the world

**Comment by FantasticFwoosh** [comment]
Is it more economic to compete in a women's workplace as a man deliberately registered and labelled as a woman?
> Macho Ma'am Randy Savage coming through

**Pull request merged**